package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "lesson_5_tests",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
```

Comment:
Add to ur package such lib "hardhat-gas-reporter"
and check how many gas you functions use
[link for library](https://www.npmjs.com/package/hardhat-gas-reporter)